packages/cli-platform-ios/src/tools/selectFromInteractiveMode.ts

```diff
@@ -9,25 +9,25 @@ import {
 interface Args {
   scheme?: string;
   mode?: string;
-  projectInfo: IosProjectInfo;
+  info: IosInfo | undefined;
 }
 
 export async function selectFromInteractiveMode({
   scheme,
   mode,
-  projectInfo,
+  info,
 }: Args): Promise<{scheme?: string; mode?: string}> {
   let newScheme = scheme;
   let newMode = mode;
 
-  if (projectInfo.schemes.length > 1) {
-    newScheme = await promptForSchemeSelection(projectInfo);
+  if (info && info?.schemes && info.schemes.length > 1) {
+    newScheme = await promptForSchemeSelection(info.schemes);
```

Comment:
Nit: `info && info?.schemes` and `info?.schemes` are equivalent. `info &&` is unnecessary. You either write `info && info.schemes` or `info?.schemes`, not mix the two. In this specific case though, I'd probably make a temp so you don't have to do the same lookup multiple times:
```suggestion
  const schemes = info?.schemes;
  if (schemes && schemes.length > 1) {
    newScheme = await promptForSchemeSelection(schemes);
```